agents-docker-compose.yml

```diff
@@ -0,0 +1,74 @@
+services:
+  message_queue:
+    build: ./src/message_queue
+    ports:
+      - "8100:8100"
+    environment:
+      - MESSAGE_QUEUE_HOST=message_queue
+      - MESSAGE_QUEUE_PORT=8100
+    networks:
+      - ragapp-network
+
+  control_plane:
+    build: ./src/control_plane
+    ports:
+      - "8000:8000"
+    environment:
+      - CONTROL_PLANE_HOST=control_plane
```

Comment:
as this is the control plane, why do we need the `CONTROL_PLANE_` prefix?

Reply:
Because we have multiple services now so it's good to add prefix to the env variable, whether we deploy it separately or not.

Reply:
i think that pattern makes sense if we have a shared .env file (which we don't have), but what are other use cases?